### ISSUE TITLE
Calling DirectContext.makeGL() causes EXCEPTION_ACCESS_VIOLATION

### ISSUE THREAD

**Issue by 7003Mars** [JetBrains/skiko#860]
Currently using lwjgl 3.3.1 and skiko 0.7.90
```kotlin
fun main() {
    glfwInit()
    val window: Long = glfwCreateWindow(100, 100, "Title", NULL, NULL)
    if (window == NULL) throw RuntimeException("Failed to create window")
    glfwMakeContextCurrent(window)

    println("Makegl?")
    DirectContext.makeGL()
    println("Made")
}
```
Running the code from `gradle run` gives this error:
```
Makegl?
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x0000000000000000, pid=12052, tid=8304
#
# JRE version: OpenJDK Runtime Environment Corretto-16.0.2.7.1 (16.0.2+7) (build 16.0.2+7)
# Java VM: OpenJDK 64-Bit Server VM Corretto-16.0.2.7.1 (16.0.2+7, mixed mode, sharing, tiered, compressed oops, compressed class ptrs, g1 gc, windows-amd64)
# Problematic frame:
# C  0x0000000000000000
#
# No core dump will be written. Minidumps are not enabled by default on client versions of Windows
#
# An error report file with more information is saved as:
# C:\Users\Rainbowhouse\Downloads\skikoskija\hs_err_pid12052.log
#
# If you would like to submit a bug report, please visit:
#   https://github.com/corretto/corretto-jdk/issues/
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
```
[hs_err_pid12052.log](https://github.com/JetBrains/skiko/files/14123856/hs_err_pid12052.log)
Interestingly, the latest version of Skija I tried(0.116.2) seems to work properly. If required, all code can be found [here](https://github.com/7003Mars/skikoskija)

**Comment by 7003Mars** [comment]
Issue seems to be resolved in version 0.7.93

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/SKIKO-860">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
